### ISSUE TITLE
Neural Network 🛜

### ISSUE THREAD

**Issue by Juma-creator** [Juma-creator/PaywithPi#4]
pip install tensorflow

import tensorflow as tf

from tensorflow.keras.models import Sequential

from tensorflow.keras.layers import Dense

model = Sequential([

    Dense(64, activation='relu', input_shape=(input_dim,)),

    Dense(64, activation='relu'),

    Dense(output_dim, activation='softmax')

])

model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])

model.fit(X_train, y_train, epochs=10, batch_size=32)

pip install torch

import torch

import torch.nn as nn

import torch.optim as optim

class NeuralNet(nn.Module):

    def __init__(self, input_dim, output_dim):

        super(NeuralNet, self).__init__()

        self.layer1 = nn.Linear(input_dim, 64)

        self.layer2 = nn.Linear(64, 64)

        self.output = nn.Linear(64, output_dim)

        self.relu = nn.ReLU()

        self.softmax = nn.Softmax(dim=1)

    def forward(self, x):

        x = self.relu(self.layer1(x))

        x = self.relu(self.layer2(x))

        x = self.softmax(self.output(x))

        return x

model = NeuralNet(input_dim, output_dim)

from tensorflow.keras.layers import ELU, LeakyReLU

model = Sequential([

    Dense(64, input_shape=(input_dim,)),

    ELU(alpha=1.0),

    Dense(64),

    LeakyReLU(alpha=0.1),

    Dense(output_dim, activation='softmax')

])

class AdvancedNeuralNet(nn.Module):

    def __init__(self, input_dim, output_dim):

        super(AdvancedNeuralNet, self).__init__()

        self.layer1 = nn.Linear(input_dim, 64)

        self.elu = nn.ELU()

        self.layer2 = nn.Linear(64, 64)

        self.leaky_relu = nn.LeakyReLU()

        self.output = nn.Linear(64, output_dim)

        self.softmax = nn.Softmax(dim=1)

    def forward(self, x):

        x = self.elu(self.layer1(x))

        x = self.leaky_relu(self.layer2(x))

        x = self.softmax(self.output(x))

        return x

model = AdvancedNeuralNet(input_dim, output_dim)
